js/tweek-client/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import 'isomorphic-fetch';
+
+export * from './types';
+export { default as TweekClient } from './TweekClient';
```

Comment:
export TweekClient should be default for backward compatibility

Reply:
it wasn't default before:
https://github.com/Soluto/tweek-clients/blob/3594b478279311940499e98e32716a875a663943/js/tweek-client/index.ts